Sorting Algorithms/Empty.txt

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
Can you change this `hello.txt` file to a `README.md` file describing all the algorithms that you added?